app/src/main/kotlin/com/hedvig/app/feature/claims/ui/commonclaim/CommonClaimActivity.kt

```diff
@@ -90,3 +91,8 @@ class CommonClaimActivity : AppCompatActivity(R.layout.activity_common_claim) {
       }
   }
 }
+
+private fun CommonClaimsData.itemTypeFromId() = when (id) {
+  "6" -> "PHONE"
```

Comment:
So 6 has some significance on the backend I presume right?

Reply:
Yeah its the id for broken phone common claim card